presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveDynamicPartitionPruning.java

```diff
@@ -86,7 +86,7 @@ protected Session getSession()
         return Session.builder(super.getSession())
                 .setSystemProperty(JOIN_REORDERING_STRATEGY, NONE.name())
                 .setSystemProperty(JOIN_DISTRIBUTION_TYPE, PARTITIONED.name()) // Avoid node local DF
-                .setSystemProperty(DYNAMIC_FILTERING_RANGE_ROW_LIMIT_PER_DRIVER, "100000")
+                .setSystemProperty(ENABLE_LARGE_DYNAMIC_FILTERS, "true")
```

Comment:
Please `io.prestosql.testing.DistributedQueryRunner.Builder#setSingleExtraProperty` instead of relying on DF defaults. Please add a comment why we need to change property in first place

Reply:
Explicitly added `dynamic-filtering.large-partitioned.max-distinct-values-per-driver` to query runner and added comments